src/components/common/ServerHeader.tsx

```diff
@@ -124,7 +126,17 @@ export default observer(({ server }: Props) => {
                         </svg>
                     </Tooltip>
                 ) : undefined}
-                <div className="title">{server.name}</div>
+                <a
+                  className="desc"
+                  onClick={() =>
+                      openScreen({
+                         id: "server_info",
+                         server,
+                     })
+                  } 
+                  style="cursor: pointer; color: var(--foreground)">
+                    <ServerName>{server.name}</ServerName>
```

Comment:
Where does `<ServerName />` come from?

Reply:
Also it may be a better idea to have the link on the inside, to avoid any potential issues in the future with layout.
Also inline styles should be removed.

Reply:
Not sure on that second point - are you referring to putting the link inside a <div> of some kind? Others can be fixed quite easily

Reply:
Swapping the order of `<ServerName>` and `<a>`, so `<ServerName><a>`.